### PR TITLE
v5.17.19

### PR DESCRIPTION
**Pull request by DanailH**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,30 @@
...
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.17.19
+
+_Jan 13, 2023_
+
+We'd like to offer a big thanks to the 3 contributors who made this release possible. Here are some highlights ✨:
+
+- 🐞 Bugfixes
+
```
Comment (MBilalShafi) [inline CHANGELOG.md:13]:
```suggestion
- 🌍 Add Belarusian (be-BY) and Urdu (ur-PK) and Improve Spanish (es-ES) locales

```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,30 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.17.19
+
+_Jan 13, 2023_
+
+We'd like to offer a big thanks to the 3 contributors who made this release possible. Here are some highlights ✨:
+
+- 🐞 Bugfixes
```
Comment (flaviendelangle) [inline CHANGELOG.md:12]: We can add the locales here

**Review by flaviendelangle — APPROVED** [review]

**Review by flaviendelangle — APPROVED** [review]

**Review by MBilalShafi — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,31 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.17.19
+
+_Jan 13, 2023_
```
Comment (LukasTy) [inline CHANGELOG.md:8]:
```suggestion
_Jan 16, 2023_
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,31 @@
...
+- [DataGridPremium] Fix Excel export not working with date strings (#7478) @cherniavskii
+- [DataGridPro] Fix missing column headers border with top-pinned rows (#7399) @cherniavskii
+
+### `@mui/x-date-pickers@v5.0.14` / `@mui/x-date-pickers-pro@v5.0.14`
+
+#### Changes
+
+- [pickers] Add Belarusian (be-BY) locale (#7450) @volhalink
+- [pickers] Add Urdu (ur-PK) locale (#7449) @MBilalShafi
+- [pickers] Improve Spanish (es-ES) locale (#7438) @Anderssxn
```
Comment (LukasTy) [inline CHANGELOG.md:29]: It's a DataGrid locale improvement PR. 🙈

**Pull request merged**